### PR TITLE
Update to API v9.

### PR DESCRIPTION
**Pull request by byte-warlock**
Tested sprint/tincture queueing.
Tested ground target queuing.
Tested Macro queueing.
Tested UI/Model mouseover, Self targeting, and Focus targeting.

### CODE REVIEW

File: Redirect/PluginUI.cs [hunk 1]
```diff
@@ -176,11 +176,11 @@ class PluginUI : IDisposable {
...
         }
 
-        private TextureWrap? FetchTexture(ushort id) {
-            Icons.TryGetValue(id, out TextureWrap? texture);
+        private IDalamudTextureWrap? FetchTexture(ushort id) {
+            Icons.TryGetValue(id, out IDalamudTextureWrap? texture);
 
-            if (texture is null && id > 0) {
-                texture = Plugin.DataManager.GetImGuiTextureIcon(id);
+            if (texture is null && id > 0) {;
```
Comment (rvaidya) [inline Redirect/PluginUI.cs:182]:
```suggestion
            if (texture is null && id > 0) {
```
Is this semicolon supposed to be here?

**Comment by aledipas** [comment]
@cairthenn

**Comment by Xorag** [comment]
I'm trying to build this branch to test it myself but running into some compiler errors, not sure why. I might be missing a dependency?
image

**Comment by mnn** [comment]
@Xorag I haven't tried it, but are you sure you are compiling [correct branch](https://github.com/byte-warlock/Redirect/tree/api9)? In changes in this PR I see some of those reported as errors in your screenshot (e.g. GameHooks) updated correctly here to use interfaces (`I*`).

**Comment by Xorag** [comment]
> @Xorag I haven't tried it, but are you sure you are compiling [correct branch](https://github.com/byte-warlock/Redirect/tree/api9)? In changes in this PR I see some of those reported as errors in your screenshot (e.g. GameHooks) updated correctly here to use interfaces (`I*`).

Yea I don't usually use visual studio and totally didn't realize i was on the wrong branch of that repo... thanks it does build now

**Comment by cairthenn** [comment]
Sorry, I haven't had any time until the weekend. 😅 I merged this, making a few adjustments to accommodate for logging & texture changes, and then I'll PR it to Dalamud.

Thanks for the help.

**Pull request merged**